hudi-common/src/main/avro/HoodieRollbackMetadata.avsc

```diff
@@ -38,14 +38,6 @@
                     "type": "long",
                     "doc": "Size of this file in bytes"
                 }
-            }], "default":null },
```

Comment:
this may not be backwards compatible while reading rollback metadata written w/ 0.10.0 or previous versions.

Reply:
I've researched this a bit since my intuition was telling that fields removal is BWC change in Avro (since Avro also supports _forward compatibility_, when you essentially read data with old schema that was produced with a new one, which is essentially the same case)

And seems like deletion is BWC in Avro:
https://docs.confluent.io/platform/current/schema-registry/avro.html

Reply:
This is an excerpt from Avro's docs:
https://avro.apache.org/docs/1.7.7/spec.html#Schema+Resolution
<img width="1339" alt="Screen Shot 2022-01-31 at 5 42 02 PM" src="https://user-images.githubusercontent.com/428277/151900978-a62c84d6-3bb7-4723-bdfd-c8d937689c34.png">

Reply:
hmmm, interesting. Can you try this out explicitly. write an avro using master branch. and then try to read it using this branch. or you can just try it out using a stand alone java main class too, which ever works. 
just wanted to ensure we are good here.